v3/tutorials/01-create-your-first-substrate-chain/index.mdx

```diff
@@ -385,9 +380,7 @@ To transfer funds to an account:
 
 After a successful transfer, you can continue to explore the front-end template components or
 stop the local Substrate node the state changes you made.
-With `--tmp` flag specified when running the Node Template, stopping the local node stops the
-blockchain and purge all persistent block data that goes with it so you can start with a clean new
-state next time you start again.
+Because you specified the `--dev` option when you started the node, stopping the local node stops the blockchain and purges all persistent block data so that you can start with a clean state next time you start the node.
```

Comment:
https://github.com/paritytech/substrate/pull/9938
We want to maybe surface this? As some users may depend on this behavior. 

This should not live in tutorials... but where? Some FAQ or Troubleshooting? 
Or rely on upstream release notes?